packages/web-react/src/components/Card/__tests__/Card.test.tsx

```diff
@@ -0,0 +1,51 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
+import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+import Card from '../Card';
+
+describe('Card', () => {
+  classNamePrefixProviderTest(Card, 'Card');
+
+  stylePropsTest(Card);
+
+  restPropsTest(Card, 'article');
+
+  it('should render card component', () => {
```

Comment:
**suggestion (if-minor):** This kind of test is repeated. Can we just create something similar to the `restPropsTest` and use it?